readme.md

```diff
@@ -280,6 +280,10 @@ The following color models can be used:
 - [`ansi`](https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit) - Example: `chalk.ansi(31).bgAnsi(93)('red on yellowBright')`
 - [`ansi256`](https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit) - Example: `chalk.bgAnsi256(194)('Honeydew, more or less')`
 
+## Browser support
+
+Since Chrome 69 ANSI escape codes are natively supported in the terminal.
```

Comment:
I would re-read this sentence. It's not correct.

Reply:
indeed